.golangci.yml

```diff
@@ -41,17 +36,16 @@ linters:
     - ineffassign
     - misspell
     - staticcheck
-    - structcheck
     - typecheck
     - unparam
     - unused
-    - varcheck
     - exportloopref
     - promlinter
 
 linters-settings:
   errcheck:
-    exclude: ./.errcheck_excludes.txt
+    exclude-functions:
+      - ./.errcheck_excludes.txt
```

Comment:
Looks like this can no longer be a txt file, but actually the contents of the file in YAML list form,
      exclude-functions:
        - (github.com/go-kit/log.Logger).Log
        - fmt.Fprintln
        - fmt.Fprint

Reply:
As per https://golangci-lint.run/usage/linters/#errcheck